### PR TITLE
Generate an enhanced manifest for OSGi support

### LINKED ISSUE #390: Add OSGi metadata to javaparser-core
It would be great if you could add OSGi metadata to javaparser-core to make it simpler to reuse the code in OSGi environments.

A good start would be exporting everything except `com.github.javaparser.ast.internal` with version 0.0.0 - assuming you don't want to do fine-grained OSGi versioning for now.

### PR DESCRIPTION
**Pull request by rombert**
Fixes #390

### CODE REVIEW

**Comment by matozoid** [comment]
All I can say is: it does not seem to break anything, so I like it.
@ftomassetti , @DeepSnowNeeL - have anything to add?

File: javaparser-core/pom.xml [hunk 1]
```diff
@@ -107,7 +107,31 @@
                     </dependency>
                 </dependencies>
             </plugin>
+            <!-- Generate an OSGi-enabled MANIFEST during the build -->
+            <plugin>
+                <groupId>biz.aQute.bnd</groupId>
+                <artifactId>bnd-maven-plugin</artifactId>
+                <version>3.2.0</version>
```
Comment (DeepSnowNeeL) [inline javaparser-core/pom.xml]: Maybe put the version in a variable like other plugins ?

Comment (matozoid) [inline javaparser-core/pom.xml]: Hm, good for consistency.

Comment (rombert) [inline javaparser-core/pom.xml]: OK, I'll add that. Do you prefer me to push a second commit with the fix or squash the existing one?

**Comment by rombert** [comment]
I've added the plugin version to parent pom, feel free to squash when merging.

**Comment by matozoid** [comment]
Thanks, will piggy back this on version 2.5.1!

**Comment by rombert** [comment]
Great, thank you!

**Pull request merged**